amnestywebsite/humanity-theme
# Leverage colour scheme from theme.json

Ref: https://github.com/amnestywebsite/humanity-theme/issues/335

We currently define a colour scheme in `theme.json`, but do not make use of the CSS vars generated by it.

This PR replaces our SCSS colour variables with the CSS vars. It also removes some unused colours within theme.json, and names them more clearly.

**Steps to test**:
1. checkout the branch
2. cd into the `./private` directory
3. run `yarn build`
4. check the following:
   - sections have grey backgrounds
   - the tweet action block alternate style uses the Twitter brand colour for its button bg
   - button colours are correct (default is primary colour; dark is... dark)
   - news index filters and list container have grey backgrounds
   - hero text background colours are correct
   - etc.
